config/src/vespa/config/frt/frtconnectionpool.cpp

```diff
@@ -97,7 +97,7 @@ namespace {
  * @return the hash value
  */
 int hashCode(const vespalib::string & s) {
```

Comment:
I suggest changing signature to return uint32_t and remove the std::abs that are lying around.